modules/ROOT/pages/tutorial/tutorial-sso-configuration.adoc

```diff
@@ -300,6 +324,20 @@ dbms.security.oidc.azure.authorization.group_to_role_mapping= "managers" = admin
 dbms.security.oidc.azure.claims.groups=roles
 ----
 
+[role=label--new-5.24]
+. (Optional). If you want to mandate that users exist in the database in order to authenticate and authorize, you can use xref:authentication-authorization/auth-providers.adoc[Auth Providers] to achieve this.
+Set the `dbms.security.require_local_user` configuration setting to `true` in the _neo4j.conf_ file to enable this mode.
+For example to create a user `jake` who can authenticate and authorize using Entra, you can use the following Cypher query:
+[source, cypher, role=noplay]
+----
+CREATE USER jake
+SET HOME DATABASE 'jakesHomeDb'
+SET AUTH 'oidc-azure' {SET ID 'jakesUniqueAzureUserId'} // `jakesUniqueAzureUserId` must match the value of the claim that you configured via dbms.security.oidc.azure.claims.username
+----
```

Comment:
I'm guessing this was the example that was inconsistent and just got updated but the comment above it says `Entra` and then the example `Azure` so I guess you might have picked the wrong one to make it consistent with XD

Though looking at some of the other examples in this Entra section all seems to use azure in them so :shrug:

Reply:
changed this one too, thanks. It still leaves us with a bit of a mixture of azure and entra. Most of the dialogue has been updated to Entra, but not much of the config in the example. I'll make a DoC to unify as much as possible.